pkg/pipeline/params/params.go

```diff
@@ -109,6 +124,7 @@ func GetPipelineParams(conf *config.Config, request *livekit.StartEgressRequest)
 			RoomId:   request.RoomId,
 			Status:   livekit.EgressStatus_EGRESS_STARTING,
 		},
+		GstReady: make(chan struct{}),
```

Comment:
Do we need this initialization both here and in GetPipelineParams? Could this lead to races where on waits on the wrong channel?